tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/Common/CommonSteps.cs

```diff
@@ -21,10 +22,19 @@ public void AResponseIsReturned(int code)
         }
 
         [Then(@"the string value of (.*) is (.*)")]
-        public async Task ThenTheStringValueOfNameIs(string section, string value)
+        public async Task ThenTheStringValueIs(string section, string value)
         {
             var content = await _response.ReadBody();
-            content.Value<string>(section).Should().Be(value);
+
+            var token = content.SelectToken(section) as JValue;
```

Comment:
In the Buying Catalogue API we added some additional checking here to make sure the section is contained within the response. It improved the test failure error message as before we would just get an exception.

Here is an example:

`
content.Should().NotBeNull();
content.ToString().Should().Contain($"\"{section}\"");
`